californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfigDefaults.java

```diff
@@ -57,12 +60,27 @@
 	public static final int DEFAULT_BLOCKWISE_STATUS_LIFETIME = 5 * 60 * 1000; // 5 mins
 
 	/**
+	 * The default MID tracker.
+	 * 
+	 * Supported values are {@code NULL}, {@code GROUPED}, or {@code HASHED}.
+	 * <p>
+	 * The default value is {@code GROUPED}.
+	 */
+	public static final String DEFAULT_MID_TRACKER = "GROUPED";
+
+	/**
 	 * The default number of MID groups.
 	 * <p>
-	 * The default value is 16.
-	 * @see GroupedMessageIdTracker
+	 * Used for {@link GroupedMessageIdTracker}. The default value is 16.
+	 */
+	public static final int DEFAULT_MID_TRACKER_GROUPS = 16;
+
+	/**
+	 * The default exchange lifetime in milliseconds.
+	 * <p>
+	 * The default value is 247s.
 	 */
-	public static final int DEFAULT_MID_PROVIDER_GROUPS = 16;
+	public static final long DEFAULT_EXCHANGE_LIFETIME = 247 * 1000;
```

Comment:
I have often thought about making the `EXCHANGE_LIFETIME` a derived value from the other config params since the way we currently do it leaves a lot of room for mis-configuration ...